wagtail/admin/templates/wagtailadmin/home.html

```diff
@@ -12,10 +12,10 @@
 {% endblock %}
 
 {% block content %}
-    <header class="merged nice-padding">
+    <header class="merged header--with-padding nice-padding">
         <div class="avatar"><img src="{% avatar_url user %}" alt="" /></div>
 
-        <div>
+        <div class="sm:w-ml-4">
```

Comment:
This is the only header that needs this special padding. It's the ideal use case for a utility class